CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,14 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2021-09-08 Notes View UI Improvements
+<p>New features:
+<ul>
+<li><p>Replaced edit pop up with direct note editable text fields</p></li>
+<li><p>Right click note to add and remove selected graph elements</p></li>
+<li><p>Delete note confirmation pop up</p></li>
+</ul>
+
```

Comment:
The first `<p>` is missing a `</p>`. The list items themselves also don't need to be contained in `<p></p>`